.ci/packaging.groovy

```diff
@@ -233,8 +233,12 @@ def runE2ETestForPackages(){
 
 def release(){
   withBeatsEnv(){
-    dir("${env.BEATS_FOLDER}") {
-      sh(label: "Release ${env.BEATS_FOLDER} ${env.PLATFORMS}", script: 'mage package')
+    withEnv([
+      "DEV=true"
+    ]) {
+      dir("${env.BEATS_FOLDER}") {
+        sh(label: "Release ${env.BEATS_FOLDER} ${env.PLATFORMS}", script: 'mage package')
+      }
```

Comment:
I am not completely familiar with the CI here, would that run `DEV=true mage package` for all the beats, quick grep through the code I don't think it would have any impact.

Reply:
The `release()` method is called for linux/Mac release: https://github.com/elastic/beats/pull/21645/files#diff-bdaa5a61de14efda6eb2bbb2a38fa9a8R93-R148

Reply:
`DEV` doesn't affect `mage package` AFAIK. Does it affect something else?

You can produce "snapshot" artifacts from packaging with `SNAPSHOT=true` if that's what you're looking for.

Reply:
@ph are we ok if we use `SNAPSHOT` instead?

Reply:
Any other concern about @michalpristas's comment? We'd like to merge this to fix some of the errors we see at the e2e tests